app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+
+  def create
+    photo = Photo.find(params[:photo_id])
+    if current_user.has_liked?(photo)
```

Comment:
I like how you've created a custom method here, but could you do something on the line below for finding likes by user in the photo model? 

`photo.find_likes(current_user)`